homeassistant/components/home_connect/manifest.json

```diff
@@ -7,5 +7,6 @@
   "documentation": "https://www.home-assistant.io/integrations/home_connect",
   "iot_class": "cloud_push",
   "loggers": ["homeconnect"],
-  "requirements": ["aiohomeconnect==0.12.1"]
+  "requirements": ["aiohomeconnect==0.12.1"],
+  "single_config_entry": true
```

Comment:
I think the reason to do this is mostly so that the user can't set up multiple config entries that have the same appliances. That would cause errors due to conflicts with the unique ids.

Reply:
Should we not use this then?

Reply:
We should add this, since using more than one config entry would error currently.